cmd/livepeer/livepeer.go

```diff
@@ -242,6 +242,10 @@ func main() {
 	if *transcoder {
 		core.WorkDir = *datadir
 		if *nvidia != "" {
+			err := core.TestNvidiaTranscoder(*nvidia)
```

Comment:
Can we make this optional via flag? (But default to true.) Something like, `-testTranscoder`